projects/packages/search/src/class-stats.php

```diff
@@ -14,6 +14,10 @@
  * Search stats (e.g. post count, post type breakdown)
  */
 class Stats {
+	const CACHE_EXPIRY             = 5 * 60; // Five minutes in seconds.
```

Comment:
Could use `MINUTE_IN_SECONDS` and drop the comment?

Reply:
The `MINUTE_IN_SECONDS` constant doesn't exist in the search package and has not yet been used here. For now, I went ahead and added it as a class constant for use in the cache expiry constant.